### ISSUE TITLE
[Bug]: power.device_battery_ina_address doesn't provide voltage to device metrics

### ISSUE THREAD

**Issue by RicInNewMexico** [meshtastic/firmware#2867]
### Category

Other

### Hardware

DIY

### Firmware Version

2.2.10

### Description

Functioning INA260 in-use as a voltage / current monitor does not provide voltage for device telemetry, but does correctly report voltage / current via the environmental telemetry module.

Device: Xiao BLE DIY, i2c bus on pins 4/5. 
Sensor Used: Adafruit INA260 breakout (Adafruit P/N 4226)

### Relevant log output

```Shell
//\ E S H T /\ S T / C

DEBUG | ??:??:?? 2 Filesystem files:
DEBUG | ??:??:?? 2  db.proto (557 Bytes)
DEBUG | ??:??:?? 2  channels.proto (88 Bytes)
DEBUG | ??:??:?? 3  module.proto (88 Bytes)
DEBUG | ??:??:?? 3  config.proto (92 Bytes)
DEBUG | ??:??:?? 3 Using analog input 32 for battery level
INFO  | ??:??:?? 3 Scanning for i2c devices...
DEBUG | ??:??:?? 3 Scanning for i2c devices on port 1
DEBUG | ??:??:?? 3 I2C device found at address 0x40
DEBUG | ??:??:?? 3 Wire.available() = 2
DEBUG | ??:??:?? 3 Register MFG_UID: 0x5449
INFO  | ??:??:?? 3 INA260 sensor found at address 0x40
INFO  | ??:??:?? 3 1 I2C devices found
DEBUG | ??:??:?? 3 acc_info = 0
DEBUG | ??:??:?? 3 found i2c sensor meshtastic_TelemetrySensorType_INA260
INFO  | ??:??:?? 3 Meshtastic hwvendor=36, swver=2.2.10.4e3576ae
DEBUG | ??:??:?? 3 Reset reason: 0x0
DEBUG | ??:??:?? 3 Setting random seed 2079015547
INFO  | ??:??:?? 3 Initializing NodeDB
INFO  | ??:??:?? 3 Loading /prefs/db.proto
INFO  | ??:??:?? 3 Loaded saved devicestate version 22
INFO  | ??:??:?? 3 Loading /prefs/config.proto
INFO  | ??:??:?? 3 Loaded saved config version 22
INFO  | ??:??:?? 3 Loading /prefs/module.proto
INFO  | ??:??:?? 3 Loaded saved moduleConfig version 22
INFO  | ??:??:?? 3 Loading /prefs/channels.proto
INFO  | ??:??:?? 3 Loaded saved channelFile version 22
INFO  | ??:??:?? 3 No /oem/oem.proto preferences found
DEBUG | ??:??:?? 3 cleanupMeshDB purged 0 entries
DEBUG | ??:??:?? 3 Expanding short PSK #1
WARN  | ??:??:?? 3 User provided a too short AES128 key - padding
INFO  | ??:??:?? 3 Wanted region 1, using US
DEBUG | ??:??:?? 3 region=1, NODENUM=0x8cb75d70, dbsize=6
DEBUG | ??:??:?? 3 WANT GPS=1
DEBUG | ??:??:?? 3 NeighborInfoModule is disabled
INFO  | ??:??:?? 3 External Notification Module Disabled
WARN  | ??:??:?? 3 SX126xInterface(cs=0, irq=1, rst=3, busy=2)
DEBUG | ??:??:?? 3 SX126X_DIO3_TCXO_VOLTAGE defined, using DIO3 as TCXO reference voltage at 1.800000 V
INFO  | ??:??:?? 3 Starting meshradio init...
DEBUG | ??:??:?? 3 (bw=250, sf=9, cr=4/8) packet symLen=2 ms, payloadSize=0, time 61 ms
DEBUG | ??:??:?? 3 (bw=250, sf=9, cr=4/8) packet symLen=2 ms, payloadSize=253, time 983 ms
INFO  | ??:??:?? 3 Radio freq=910.375, config.lora.frequency_offset=0.000
INFO  | ??:??:?? 3 Set radio: region=US, name=PRIMARY, config=4, ch=33, power=30
INFO  | ??:??:?? 3 Radio myRegion->freqStart -> myRegion->freqEnd: 902.000000 -> 928.000000 (26.000000 mhz)
INFO  | ??:??:?? 3 Radio myRegion->numChannels: 104 x 250.000kHz
INFO  | ??:??:?? 3 Radio channel_num: 34
INFO  | ??:??:?? 3 Radio frequency: 910.375000
INFO  | ??:??:?? 3 Slot time: 42 msec
INFO  | ??:??:?? 3 Set radio: final power level=22
INFO  | ??:??:?? 3 SX126x init result 0
INFO  | ??:??:?? 3 Frequency set to 910.375000
INFO  | ??:??:?? 3 Bandwidth set to 250.000000
INFO  | ??:??:?? 3 Power output set to 22
DEBUG | ??:??:?? 3 Current limit set to 140.000000
DEBUG | ??:??:?? 3 Current limit set result 0
DEBUG | ??:??:?? 3 Setting DIO2 as RF switch
DEBUG | ??:??:?? 3 Using MCU pin 7 as RXEN and pin -1 as TXEN to control RF switching
INFO  | ??:??:?? 3 Set RX gain to boosted mode; result: 0
INFO  | ??:??:?? 3 SX1262 Radio init succeeded, using SX1262 radio
DEBUG | ??:??:?? 3 (bw=250, sf=9, cr=4/8) packet symLen=2 ms, payloadSize=237, time 926 ms
```

**Comment by garthvh** [comment]
It should go in the environment metrics

**Comment by RicInNewMexico** [comment]
It should when it's being used for that use-case, but the entire point of being able to set "power.device_battery_ina_address" to the proper i2c bus address is so it can be used for battery level monitoring in devices that use an i2c voltage sensor in place of an ADC voltage divider.

**Comment by garthvh** [comment]
As is it is meant to be a separate data source, mostly for monitoring of a solar system.  The current code has zero functionality to replace the existing device metrics with the INA.

**Comment by RicInNewMexico** [comment]
I'm just logging it as a bug, related to [ PR #2536](https://github.com/meshtastic/firmware/pull/2536), because I was asked to..

**Comment by thebentern** [comment]
I think this is fixed in #3002. Closing
